src/components/DroughtHistory.vue

```diff
@@ -105,18 +102,11 @@ export default {
 
     // Define scrollTo dates
     this.scrollToDates = [
-      {id: '1920s', date: '1921-10-01'}, 
-      {id: '1930s', date: '1930-01-01'}, 
-      {id: '1940s', date: '1940-01-01'}, 
-      {id: '1950s', date: '1950-01-01'}, 
-      {id: '1960s', date: '1960-01-01'},
-      {id: '1970s', date: '1970-01-01'},
-      {id: '1980s', date: '1980-01-01'},
-      {id: '1990s', date: '1990-01-01'},
-      {id: '2000s', date: '2000-01-01'},
-      {id: '2010s', date: '2010-01-01'}]
-    // Get all possible button ids
-    this.decadeIDs = this.scrollToDates.map(scrollDate => scrollDate.id);
+      {id: '1930', name: 'Dust Bowl', start: '1930-02-01', end: '1941-08-31'}, 
```

Comment:
Nice to have idea or maybe an issue: Would it work to get these dates from a target table so that the R-based viz and the `vue` match these dates automatically?

Reply:
ooh interesting thought. we could export a csv from the pipeline, either directly to an assets folder or to S3, and then pull it in and read it.